src/components/StatementOfCashFlow/StatementOfCashFlow.tsx

```diff
@@ -88,6 +109,11 @@ const StatementOfCashFlowView = ({
         }
         dateFormat='MMM d'
         mode={datePickerMode}
+        allowedModes={['dayRangePicker', 'monthPicker']}
```

Comment:
Minor: I'd love to swap the order here so it defaults to the first one and the 2nd looks like an optional advanced mode, which it is.